integ/run-integ.ps1

```diff
@@ -114,6 +114,10 @@ Function Install-Package {
         Invoke-WebRequest "https://github.com/docker/compose/releases/download/v${DockerComposeVersion}/docker-compose-Windows-x86_64.exe" -UseBasicParsing -OutFile $Env:ProgramFiles\Docker\docker-compose.exe
     }
     Write-Host $( docker-compose version )
+
+    # build validator images
+    docker build -t $env:CW_INTEG_VALIDATOR_IMAGE --file "${IntegTestRoot}/validate_cloudwatch/Dockerfile" "${IntegTestRoot}/validate_cloudwatch/"
```

Comment:
After each `docker build` command, we should check if the command succeeded or not. A check as below should do it-

```
if ($LASTEXITCODE)
{
    throw ("Failed to build $env:CW_INTEG_VALIDATOR_IMAGE image")
}
```

Reply:
added